components/Quiz.tsx

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable max-len */
+
+import CorrectIcon from './icons/Correct';
+import IncorrectIcon from './icons/Incorrect';
+import { useState } from 'react';
+
+interface IconWrapperProps {
+  children: React.ReactNode
+}
```

Comment:
Consider using `PropsWithChildren` exported by React package.

Reply:
Thanks,It's ok now